src/test/util/event.rs

```diff
@@ -143,6 +143,55 @@ impl EventClient {
         }
     }
 
+    pub async fn merge_options(
+        options: Option<ClientOptions>,
+        use_multiple_mongoses: Option<bool>,
+    ) -> Self {
+        let mut options = match options {
+            Some(mut options) => {
+                options.hosts = CLIENT_OPTIONS.hosts.clone();
+                merge_options!(
+                    CLIENT_OPTIONS.clone(),
+                    &mut options,
+                    [
+                        app_name,
```

Comment:
Is it worth defining an additional macro for cases like this where we want the entirety of options? If we ever need to do this somewhere else, it would be nice to only have to maintain the list in one place.

Reply:
Is there another use case you can think of? Everywhere I've needed to do this has been in the context of merging the test options with options in spec tests which is taken care of by this method

Reply:
Fair enough. We can hold off until/unless we run into another case where we need it.